fix libffi builds on MSVC

Credit to the deno conda-forge feedstock: https://github.com/conda-forge/deno-feedstock/blob/main/recipe/01-fix-libffi-msvc.patch
---

diff --git a/Cargo.toml b/Cargo.toml
index 05d29d4..d22ab88 100644
--- a/Cargo.toml
+++ b/Cargo2.toml
@@ -3,6 +3,9 @@ resolver = "2"  # Needed for proper wgpu compilation

members = ["vl-convert-rs", "vl-convert-python"]

+[patch.crates-io]
+libffi-sys = { git = "https://github.com/tov/libffi-rs", rev = "d0704d634b6f3ffef5b6fc7e07fe965a1cff5c7b" }
+
 [profile.release]
 strip = true  # Automatically strip symbols from the binary
 opt-level = "z"  # Optimize for size
